### PR TITLE
Fixed intrahealth pillow for the short term.

### PR DESCRIPTION
**Pull request by emord**
This is not code that I'm proud of, but it gets intrahealth/ipm unblocked on their deadline and it's already running on their server

pna context: @kaapstorm @javierwilson @snopoke 
buddy @proteusvacuum

### CODE REVIEW

File: corehq/ex-submodules/fluff/pillow.py [hunk 1]
```diff
@@ -71,6 +72,13 @@ def _is_doc_type_deleted_match(self, type):
 
     def change_transform(self, doc_dict):
         delete = False
+        if self.document_class == XFormInstanceSQL:
```
Comment (emord) [inline corehq/ex-submodules/fluff/pillow.py]: This is nasty

File: corehq/ex-submodules/fluff/pillow.py [hunk 2]
```diff
@@ -20,8 +21,8 @@ def __init__(self, indicator_class, delete_filtered=False):
         self.document_class = indicator_class.document_class
         self.save_direct_to_sql = indicator_class().save_direct_to_sql
         self.deleted_types = indicator_class.deleted_types
-        self.doc_type = indicator_class.document_class._doc_type
         self.wrapper = indicator_class.wrapper or indicator_class.document_class
```
Comment (emord) [inline corehq/ex-submodules/fluff/pillow.py]: I did a quick grep and didn't see anything in custom pillows that ever used it. I'm guessing htat something old used it and it was never cleaned up

**Review by emord — COMMENTED** [review]
For a bit more context, the pillow (and fluff in general) was running into errors because it assumes that hte documents live in couch. This changes it to grab the forms/cases from postgres and then it calls `to_json` on it and then it re-wraps it with the couch doc (since there's no wrap method on sql models).

Maybe we can rewrite these document processors in ucr?

**Review by kaapstorm — APPROVED** [review]

**Review by snopoke — COMMENTED** [review]
Not ideal but OK. Are there tests for this in case a SQL model change breaks this?

**Comment by emord** [comment]
> Not ideal but OK. Are there tests for this in case a SQL model change breaks this?

Not at hte moment. I think we need to go back through fluff and get it to use FormAccessors and CaseAccessors to ensure this works well and we can migrate the other domains. There was some deadline today so I just got this working as quickly as possible for kelly and the team

**Comment by emord** [comment]
forgot about this. these aren't used anymore anyways

**Pull request closed without merging**